docs/CONFIGURING_FLAT_CONFIG.md

```diff
@@ -148,6 +148,86 @@ module.exports = tseslint.config(
 
 By setting up our config in this way, we have complete control over what rules etc apply to what file types and our separate concerns remain clearer and easier to maintain. The schematics provided by angular-eslint will already configure your project in this way.
 
+## Notes for `eslint-plugin-prettier` users
+
+Prettier is an awesome code formatter which can be used entirely independently of linting.
+
+Some folks, however, like to apply prettier by using it inside of ESLint, using `eslint-plugin-prettier`. If this applies to you then you will want to read this section on how to apply it correctly for HTML templates. Make sure you read and fully understand the information above on the importance of `"overrides"` before reading this section.
+
+If you choose to use `eslint-plugin-prettier`, **please ensure that you are using version 5.1.0 or later**, and apply the following configuration to ESLint and prettier:
+
+**.prettierrc**
+
+```json
+{
+  "overrides": [
+    {
+      "files": "*.html",
+      "options": {
+        "parser": "angular"
+      }
+    }
+  ]
+}
+```
+
+**eslint.config.js**
```

Comment:
This should have a clearer name, it seems like it is the root config, even though the contents are inheriting from the root config